metron-analytics/metron-profiler-client/src/main/java/org/apache/metron/profiler/client/stellar/GetProfile.java

```diff
@@ -91,12 +91,12 @@
   /**
    * A global property that defines the name of the column family used to store profile data.
    */
-  public static final String PROFILER_COLUMN_FAMILY = "profiler.client.column.family";
+  public static final String PROFILER_COLUMN_FAMILY = "profiler.client.hbase.column.family";
 
   /**
    * A global property that defines the name of the HBaseTableProvider implementation class.
    */
-  public static final String PROFILER_HBASE_TABLE_PROVIDER = "profiler.client.hbase.table.provider";
+  public static final String PROFILER_HBASE_TABLE_PROVIDER = "hbase.provider.impl";
```

Comment:
This isn't an objection, just a question: I'm not yet real familiar with the structure of the configuration tree in ZK.  Is this "hbase.provider.impl" parameter stored in a part of the tree where it clearly ONLY applies to profiler?  Otherwise I would prefer to leave it with a "profiler." prefix.